packages/office-ui-fabric-react/src/components/Panel/Panel.styles.ts

```diff
@@ -119,33 +208,16 @@ export const getStyles = (props: IPanelStyleProps): IPanelStyles => {
         right: 0,
         bottom: 0
       },
-      !isOpen &&
-        !isAnimating &&
-        isHiddenOnDismiss && {
-          visibility: 'hidden'
-        },
       isCustomPanel && classNames.custom,
       className
     ],
     overlay: [
       {
-        pointerEvents: 'none',
-        opacity: 1,
-        cursor: 'pointer',
-        transition: `opacity ${AnimationVariables.durationValue3} ${AnimationVariables.easeFunction1}`,
-        selectors: {
-          '@media screen and (-ms-high-contrast: active), (-ms-high-contrast: none)': {
-            // For IE high contrast mode
-            backgroundColor: 'transparent'
-          }
-        }
-      },
-      isOpen && {
-        cursor: 'pointer',
-        pointerEvents: 'auto'
```

Comment:
Unnecessary as `Overlay` will be rendered only when the Panel is open... so I moved it up.